File: src/main/java/net/hectus/neobb/turn/person_game/categorization/GameEndingCounterCategory.java
package net.hectus.neobb.turn.person_game.categorization;

import net.kyori.adventure.text.format.NamedTextColor;
import net.kyori.adventure.text.format.TextColor;
import org.bukkit.Material;

public interface GameEndingCounterCategory extends Category {
    @Override
    default TextColor categoryColor() {
        return NamedTextColor.GOLD;
    }

    @Override
    default Material categoryItem() {
        return Material.GRAY_CARPET;
    }

    @Override
    default String categoryName() {
        return "game_ending_counter";
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/person_game/categorization/GameEndingCounterCategory.kt
package net.hectus.neobb.turn.person_game.categorization

import net.kyori.adventure.text.format.NamedTextColor
import net.kyori.adventure.text.format.TextColor
import org.bukkit.Material

interface GameEndingCounterCategory : Category {
    override val categoryColor: TextColor get() = NamedTextColor.GOLD

    override val categoryItem: Material get() = Material.GRAY_CARPET
    override val categoryName: String get() = "game_ending_counter"
}
